src/models/streaming/AudioConfig.ts

```diff
@@ -0,0 +1,22 @@
+export class AudioConfig {
+    contentType: string;
+    layout?: string;
+    rate?: number;
+    format?: string;
+    channels?: number;
+    constructor(contentType = "audio/*", layout?: string, rate?: number, format?: string, channels?: number) {
```

Comment:
minor space
```suggestion
    
    constructor(contentType = "audio/*", layout?: string, rate?: number, format?: string, channels?: number) {
```

Reply:
What do you think about this method of private property definition?
 https://kendaleiv.com/typescript-constructor-assignment-public-and-private-keywords/

The syntax is not popular, so it might be too confusing.

Reply:
I'm not a huge fan. It would remove some lines but I think it would make it a lot less clear to anyone who doesn't know this syntax